department-of-veterans-affairs/caseflow
# Ensure we show org tasks on a timed hold in the org queues

Resolves #12782

### Description
Previously, in an organization queue that uses pagination, there are two tabs for organization tasks that are on hold. The "Assigned" tab shows all active children of tasks assigned to the organization. The "On hold" tab shows all on hold children of tasks assigned to the organization. Naturally, we only show tasks where `hide_from_queue_table_view` is false. However, we can have organization tasks on timed holds (`NoShowHearingTask`s are put on hold the moment they are created). This would leave the users with no way to track `NoShowHearingTask`s as their children are hidden from the queue table view.

This change makes sure that we display the children along with any parents that are on a timed hold

### Acceptance Criteria
- [ ] `NoShowHearingTask`s are visible in the "Assigned" tab of queue table

### Testing Plan
1. Make a `NoShowHearingTask`

```ruby
NoShowHearingTask.create_with_hold(Appeal.first.root_task)
```
2. Log in as BVATWARNER
3. Go to the [hearings management assigned tab](http://localhost:3000/organizations/hearings-management?tab=assigned&page=1)
4. Ensure the `NoShowHearingTask` is shown

### User Facing Changes

![Screen Shot 2019-12-31 at 10 45 41 AM](https://user-images.githubusercontent.com/45575454/71626361-bba0a400-2bba-11ea-9985-c68b7f9c8532.png)
